index.js

```diff
@@ -91,7 +95,7 @@ if(data.noSecurityHotspot){
 }
 else{
   // For newer versions of sonar, rules and issues may be of type VULNERABILITY or SECURITY_HOTSPOT
-  DEFAULT_FILTER="&types=VULNERABILITY,SECURITY_HOTSPOT"
+  DEFAULT_FILTER="&types=VULNERABILITY,SECURITY_HOTSPOT,BUG,CODE_SMELL"
```

Comment:
If you want bugs and code_smells, please use argument "allbugs=true"
By default sonar-report exports only vulnerabilities because it's why it was created in the first place :)

Reply:
Good catch, I have reverted the change that was never intended to be checked in.